workflow/rules/trimming.smk

```diff
@@ -10,25 +12,48 @@ rule get_sra:
 
 def get_raw_fastq(wildcards):
     unit = units.loc[wildcards.sample].loc[wildcards.unit]
+    if unit["fq1"].endswith("gz"):
+        ending = ".gz"
+    else:
+        ending = ""
+
     if pd.isna(unit["fq1"]):
         # SRA sample (always paired-end for now)
         accession = unit["sra"]
         return expand("sra/{accession}_{read}.fastq", accession=accession, read=[1, 2])
     if pd.isna(unit["fq2"]):
         # single end local sample
-        return unit["fq1"].values
+        return f"pipe/cutadapt/{unit.sample_name}-{unit.unit_name}.fq1.fastq{ending}"
```

Comment:
Using `f` is in principle nice, but Snakemake still allows python 3.5, which does not have it. I think running this workflow with 3.5 would lead to a syntax error then. Or is it possible to check the version without running into a syntax error with this?

Reply:
ah ok, I will pay attention to this in future PR